internal/pkg/api/server/templates/service/service_test.go

```diff
@@ -1,37 +0,0 @@
-package service
-
-import (
-	"context"
-	"log"
-	"os"
-	"testing"
-
-	"github.com/stretchr/testify/assert"
-
-	"github.com/keboola/keboola-as-code/internal/pkg/api/server/templates/dependencies"
-	"github.com/keboola/keboola-as-code/internal/pkg/env"
-	"github.com/keboola/keboola-as-code/internal/pkg/model"
-)
-
-func TestCustomRepositoryPath(t *testing.T) {
-	t.Parallel()
-
-	logger := log.New(os.Stderr, "", 0)
-	envs, err := env.FromOs()
-	assert.NoError(t, err)
-
-	d, err := dependencies.NewContainer(context.Background(), "file:///var/templates", false, logger, envs)
-	assert.NoError(t, err)
-	res := repositories(d)
-	assert.Len(t, res, 1)
-	assert.Equal(t, model.RepositoryTypeDir, string(res[0].Type))
-	assert.Equal(t, "/var/templates", res[0].Url)
-
-	d, err = dependencies.NewContainer(context.Background(), "https://github.com/keboola/keboola-as-code-templates:main", false, logger, envs)
-	assert.NoError(t, err)
-	res = repositories(d)
```

Comment:
Tento test uz nema zmysel, kedze `repositories` je presunute do `dependencies` a je to uz by obycajny getter.